app/mainwindow.cpp

```diff
@@ -475,7 +475,7 @@ void MainWindow::contextMenuEvent(QContextMenuEvent *event)
     protectedMode->setCheckable(true);
     protectedMode->setChecked(m_protectedMode);
 
-    QAction * toggleSettings = new QAction(tr("Configure..."));
+    QAction * toggleSettings = new QAction(tr("Configure…"));

```

Comment:
This shouldn't get changed since the text is for `LANG=C` which will only use ASCII encoding, not for something similar to `en_US.UTF-8`. See the [`locale` manpage](https://man.openbsd.org/locale.1), [`setlocale` manpage](https://man7.org/linux/man-pages/man3/setlocale.3.html), [debian i18n page](https://www.debian.org/doc/manuals/debian-reference/ch08.en.html#_basics_of_encoding) for details.

If you would like to use `…` for UI string displaying, consider adding something like `en` or `en_US` as a translatable language and do it there?

Reply:
Would work if that is the translated language too, otherwise it fails the check between tripledot and ellipsis from source to translation.